python/cudf/cudf/core/column/categorical.py

```diff
@@ -29,7 +25,19 @@
 )
 
 if TYPE_CHECKING:
-    from cudf._typing import SeriesOrIndex, SeriesOrSingleColumnIndex
+    from collections import abc
+
+    from numba import cuda
```

Comment:
Nit: can we make this `import numba.cuda` and update the typing accordingly? I don't want this to be confused with `import cuda` from cuda-python.

Reply:
Sounds good, changed